### PR TITLE
showcase an example in readme

### PR DESCRIPTION
**Pull request by oxalorg**
both using promises and using kitchen async

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,37 @@ typically needed or wanted for a production setup. Require
...
+A simple JSON get request.
+
+``` clojure
+(require '[lambdaisland.fetch :as fetch])
+
+(-> (fetch/get "https://reqres.in/api/users/2")
+    (.then (fn [resp]
+             (-> resp
+                 :body
+                 (js->clj :keywordize-keys true)
```
Comment (plexus) [inline README.md:66]: Use of `js->clj` is generally a bad practice, we shouldn't be setting a bad example here.

Comment (oxalorg) [reply README.md]: @plexus Thanks, fixed!

**Comment by humorless** [comment]
How about also adding a `post` && `json` example like this?

```
  (let [body {:name "jack"                                                                               
                     :age 49}                                                               
          opts {:body body                                                                                
                    :method :post                                                                             
                    :content-type :json                                                                       
                    :accept :json}
           url "/api/register"]                                                                                                                                        
      (p/try                                                                                              
          (p/let [resp (fetch/request url opts)                                                 
                      data (:body resp)]                                                                    
          (tap> data))))  
```

**Comment by plexus** [comment]
LGTM!

**Pull request merged**